package.json

```diff
@@ -3,7 +3,7 @@
   "name": "mongodb-vscode",
   "displayName": "MongoDB for VS Code",
   "description": "Connect to MongoDB and Atlas directly from your VS Code environment, navigate your databases and collections, inspect your schema and use playgrounds to prototype queries and aggregations.",
-  "version": "0.5.1-dev.0",
```

Comment:
Just a thought for later .. we could automate also this, is pretty simple .. although it would be even better to just sick with `0.0.0-dev-0` if that is only cosmetic.